pkg/kosmosctl/rsmigrate/rsimport.go

```diff
@@ -0,0 +1,135 @@
+package rsmigrate
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctlutil "k8s.io/kubectl/pkg/cmd/util"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+	mcsv1alpha1 "sigs.k8s.io/mcs-api/pkg/apis/v1alpha1"
+)
+
+var importExample = templates.Examples(i18n.T(`
+		# import resource from control plane to knode
+		kosmosctl import service foo -n namespacefoo --kubecnfig=[control plane kubeconfig] --to-node knodefoo
+`))
+
+var importErr string = "kosmosctl import complete error"
+
+type CommandImportOptions struct {
+	*CommandOptions
+	DstKnodeOptions *KnodeOptions
+}
+
+// NewCmdImport import resource
+func NewCmdImport(f ctlutil.Factory) *cobra.Command {
+	o := &CommandImportOptions{
+		CommandOptions:  &CommandOptions{SrcKnodeOptions: &KnodeOptions{}},
+		DstKnodeOptions: &KnodeOptions{},
+	}
+
+	cmd := &cobra.Command{
+		Use:                   "import",
+		Short:                 i18n.T("Import resource to knode"),
+		Long:                  "",
+		Example:               importExample,
+		SilenceUsage:          true,
+		DisableFlagsInUseLine: true,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctlutil.CheckErr(o.Complete(f, cmd))
+			ctlutil.CheckErr(o.Validate(cmd))
+			ctlutil.CheckErr(o.Run(f, cmd, args))
+			return nil
+		},
+	}
+	cmd.Flags().StringVarP(&o.MasterKubeConfig, "kubeconfig", "", "", "kubeconfig")
+	cmd.Flags().StringVarP(&o.Namespace, "namespace", "n", "default", "namespace")
+	cmd.Flags().StringVar(&o.SrcKnodeOptions.KnodeName, "node", "", "import resource from this soure knode ")
+	cmd.Flags().StringVar(&o.DstKnodeOptions.KnodeName, "to-node", "", "import resource to this destination knode")
```

Comment:
node—>knode and to-node—>to-knode, what do you think?

Reply:
modified